frontend/src/plugins/core/RenderHTML.tsx

```diff
@@ -39,10 +39,29 @@ const replaceValidIframes = (domNode: DOMNode) => {
   }
 };
 
+const replaceSrcScripts = (domNode: DOMNode): JSX.Element | undefined => {
+  if (domNode instanceof Element && domNode.name === "script") {
+    // Missing src, we don't handle inline scripts
```

Comment:
Technically, I guess I might want to in `mohtml` at some point? Maybe via alpine.js or surreal.js.

Reply:
Maybe, its not hard to add, but tempted to just hold off and see why people need it, if people do, this is the change

```ts
const replaceSrcScripts = (domNode: DOMNode): JSX.Element | undefined => {
  if (domNode instanceof Element && domNode.name === "script") {
    const { src, ...attrs } = domNode.attribs;

    // Handle inline scripts
    if (!src) {
      const script = document.createElement("script");
      // Set any additional attributes
      Object.entries(attrs).forEach(([key, value]) => {
        script.setAttribute(key, value);
      });
      // Set content if any exists
      if (domNode.children?.[0]?.data) {
        script.textContent = domNode.children[0].data;
      }
      document.head.append(script);
      return <></>;
    }

    // Handle src scripts
    if (!document.querySelector(`script[src="${src}"]`)) {
      const script = document.createElement("script");
      script.src = src;
      // Set any additional attributes
      Object.entries(attrs).forEach(([key, value]) => {
        script.setAttribute(key, value);
      });
      document.head.append(script);
    }
    return <></>;
  }
};
```